# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLFramebuffer.h
# Commit: dd00e2da3a0f
# Full Hash: dd00e2da3a0f11a7cc0af9e5ece21a339f4393eb
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-11 04:04:33
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r 69cc1de6f754 -r dd00e2da3a0f dom/canvas/WebGLFramebuffer.h
--- a/dom/canvas/WebGLFramebuffer.h	Wed Jun 10 21:57:19 2020 +0000
+++ b/dom/canvas/WebGLFramebuffer.h	Wed Jun 10 22:21:02 2020 +0000
@@ -11,7 +11,6 @@
 #include "mozilla/WeakPtr.h"
 
 #include "WebGLObjectModel.h"
-#include "WebGLRenderbuffer.h"
 #include "WebGLStrongTypes.h"
 #include "WebGLTexture.h"
 #include "WebGLTypes.h"
@@ -58,11 +57,10 @@
 
   ////
 
-  WebGLFBAttachPoint() = default;
+  WebGLFBAttachPoint();
+  explicit WebGLFBAttachPoint(WebGLFBAttachPoint&);  // Make this private.
   WebGLFBAttachPoint(const WebGLContext* webgl, GLenum attachmentPoint);
 
-  explicit WebGLFBAttachPoint(WebGLFBAttachPoint&) =
-      default;  // Make this private.
 
  public:
   ~WebGLFBAttachPoint();